src/app/ark-utility.ts

```diff
@@ -0,0 +1,13 @@
+import {WALLET_UNIT_TO_SATOSHI} from "@app/app.constants";
+
+export class ArkUtility {
```

Comment:
Well I did not like this file in the app folder, you can create a `utils` folder

Reply:
done